### DISCUSSION TITLE
[request] DS18x20 temperature sensor

### DISCUSSION THREAD

**Discussion by Massimo65** [arendst/Tasmota#15972] [Q&A] [answered]
Good morning

Thanks for this fantastic software, I am installing numerous temperature sensors with DS18x20 chip, I would need to perform a calibration of the same with the TempOffset command for each single sensor. At the moment this is not possible, can it be implemented?
Thank you

**Comment by Jason2866** [comment, marked as answer]
Just my 2 cents. If you need better precision buy a different sensor OR buy more of this and select the sensors. TempOffset is NOT a calibration.

**Comment by Massimo65** [comment]
Good morning
on the page:
https://tasmota.github.io/docs/DS18x20/#tasmota-settings
is reported
tempoffset can be used for calibrating the measured temperature. This setting affects all temperature sensors on the device.
thk

**Reply by sfromis** [reply]
https://github.com/cpetrich/counterfeit_DS18B20/ basically means that there are lots of fake DS18B20 sensors out there, with very "varying quality", to put it kindly. If you care about precision, make sure to only buy from a "really reputable" source, with a higher price. Sellers claiming "genuine" are not. And then you should not need to change the number provided by the sensor.

**Comment by Jason2866** [comment]
You can set an offset. That is not really calibration. I will fix the docu :-)
